### PR TITLE
Added max heap and orderedDict info

### PR DESCRIPTION
**Pull request by syedwajid01**

### CODE REVIEW

**Review by AbdulMalikDev — CHANGES_REQUESTED** [review]
Please look at the changes and update the PR.

File: README.md [hunk 1]
```diff
@@ -222,6 +291,40 @@ heapq.heappop(nums) # Pop an element from the heap
...
+
+#All the highest numbers are now the lowest and vice versa.
+
+#Just remember that when you pop an element to multiply it with -1 in order to get the original value again.
+
+#Example: 
+
+import heapq
+heap = []
+heapq.heappush(heap, 1*(-1))
```
Comment (AbdulMalikDev) [inline README.md:312]: heapq.heappush() function already mentioned above with example. this is a duplicate.

**Review by AbdulMalikDev — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (syedwajid01) [reply README.md:312]: this heappush() is an example for maxHeap, were we are multiplying the number with -1 before pushing , prev example was for min heap.

Comment (AbdulMalikDev) [reply README.md:312]: Okay we can work with that.

File: README.md [hunk 2]
```diff
@@ -133,6 +133,71 @@ dict.update({KEY:VALUE})
...
+
+Important Points:
+
+1) If the value of a certain key is changed, the position of the key remains unchanged in OrderedDict.
+
+2) Deleting and re-inserting the same key will push it to the back as OrderedDict, however, maintains the order of insertion.
+
+Other Considerations:  
+
+* Ordered dict in Python version 2.7 consumes more memory than normal dict. This is due to the underlying Doubly Linked List implementation for keeping the order. In Python 2.7 Ordered Dict is not dict subclass, it’s a specialized container from collections module.
```
[the remark this answers is not in the document]
Comment (AbdulMalikDev) [reply README.md:193]: @syedwajid01 waiting for changes.

[the remark this answers is not in the document]
Comment (syedwajid01) [reply README.md]: @AbdulMalikDev  I removed Ordered dict info, I think normal dict is enough for Leetcode problems.

**Review by AbdulMalikDev — APPROVED** [review]

**Comment by AbdulMalikDev** [comment]
Thank you for your PR!

**Pull request merged**